R/oc_points.R

```diff
@@ -0,0 +1,111 @@
+#' List of points for opencage queries
+#'
+#' Create a list of points (latitude/longitude coordinate pairs) for opencage
+#' queries.
+#'
+#' @param latitude,longitude Numeric vectors of latitude and longitude values.
+#' @param data A \code{data.frame} containing at least 2 columns with
+#'   \code{latitude} and \code{longitude} values.
+#' @param ... Ignored.
+#'
+#' @return A \code{point_list}, i.e. a list of points. Each point is a named
+#'   vector of length 2 containing a latitude/longitude coordinate pair.
+#' @export
+#'
+#' @examples
+#' oc_points(-21.01404, 55.26077)
+#'
+#' xdf <-
+#'   data.frame(
+#'     place = c("Hamburg", "Los Angeles"),
+#'     lat = c(53.5503, 34.0536),
+#'     lon = c(10.0006, -118.2427)
+#'   )
+#' oc_points(
+#'   data = xdf,
+#'   latitude = lat,
+#'   longitude = lon
+#' )
+#' \dontrun{
+#' # create bbox list column with dplyr
```

Comment:
Could you explain this example a bit more. I am not sure I understand how it works.

Reply:
It is equivalent to the example in `oc_bbox()`. I.e. it creates a `point_list` from a `data.frame` with a column with latitude and longitude values respectively. 
But maybe it was the following example that threw you off?